recipes/fcd_torch/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you make a PR to add this filename to the `MANIFEST.in` upstream?

Reply:
done at https://github.com/rynkl/fcd-torch/pull/1